### PR TITLE
HTTPS links to Vimeo embeds

### PR DESCRIPTION
**Pull request by coliff**
This PR changes the iframe links to load over HTTPS. This is what Vimeo.com themselves recommends.

### CODE REVIEW

**Comment by bep** [comment]
Hey, and thanks for this. A lesson learned from last round of similar template change: We have tests that tests the output of these templats (not sure about this one) and we need to run `mage generate` to see if they passes or not. So I cannot merge this before those are generated.

**Comment by coliff** [comment]
ah OK. I'll take another look at getting `merge generate` working again.

File: tpl/tplimpl/embedded/templates/shortcodes/vimeo.html [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
-  <iframe src="//player.vimeo.com/video/{{ .Get "id" }}" {{ if not (.Get "class") }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+  <iframe src="https://player.vimeo.com/video/{{ .Get "id" }}" {{ if not (.Get "class") }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
  </div>{{ else }}
 <div {{ if len .Params | eq 2 }}class="{{ .Get 1 }}"{{ else }}style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;"{{ end }}>
-  <iframe src="//player.vimeo.com/video/{{ .Get 0 }}" {{ if len .Params | eq 1 }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+  <iframe src="https://player.vimeo.com/video/{{ .Get 0 }}" {{ if len .Params | eq 1 }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```
Comment (XhmikosR) [inline tpl/tplimpl/embedded/templates/shortcodes/vimeo.html]: Nitpicking but while at it maybe add a space after `:` in border for consistency :)

Comment (coliff) [reply tpl/tplimpl/embedded/templates/shortcodes/vimeo.html]: well spotted - will do :-)

**Comment by XhmikosR** [comment]
This is now redundant @ bep after #6281

**Comment by bep** [comment]
@coliff thanks, anyway. There are plenty of other work to do ...

**Pull request closed without merging**